packages/types/src/type/Vote.ts

```diff
@@ -2,41 +2,98 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { isBoolean } from '@polkadot/util';
+import { isBoolean, isNumber, isObject } from '@polkadot/util';
 
+import Conviction from './Conviction';
+import Struct from '../codec/Struct';
+import U8a from '../codec/U8a';
+import Boolean from '../primitive/Bool';
 import I8 from '../primitive/I8';
+import { AnyJsonObject } from '../types';
 
 /**
  * @name Vote
  * @description
  * A number of lock periods, plus a vote, one way or the other.
  */
-export default class Vote extends I8 {
+export default class Vote extends U8a {
+  private _aye: Boolean | I8;
+  private _conviction?: Conviction;
```

Comment:
I think these 2 are not necessary.

I would add 2 getters in this class:
- `get aye (): Boolean`
- `get conviction (): Conviction`